src/compiler/repositories.md

```diff
@@ -0,0 +1,25 @@
+# Repositories maintained by the Compiler Team
+
+The obvious focus of the Compiler Team is the [rustc] compiler itself, but the team also maintains a few other repositories:
+
+- An experimental [C codegen backend][c-backend] for rustc, which lowers Rust MIR to C code and compiles it with a C compiler. Currently maintained by X and Y.
+- A [Cranelift][cranelift-backend] experimental rustc backend, based on [Cranelift]. This has the potential to improve compilation times in debug mode. Currently maintained by X and Y.
+- A Rust port of the [LLVM::APFloat][rustc_apfloat] library. Maintained by X abnd Y.
+- A fork of [Enzyme], a high-performance automatic differentiator of LLVM and MLIR (more info at this [link][enzyme-mit]). The fork is currently maintained by X and Y.
+- [Thorin], a DWARF packaging utility supporting GNU extension and DWARF 5 package formats. Maintained by X and Y.
+- The [Rust Forge][forge], the documentation website you're reading. Maintained by the Rust project.
```

Comment:
It seems like the GCC backend is missing from this list.

Reply:
We as T-compiler have "only" `write` permissions on that repo (see [list](https://github.com/orgs/rust-lang/teams/compiler/repositories)). Do you think this is that should be changed? Or maybe just add the repo to the list even if we don't have `mantain` role?

Reply:
I'll let @antoyo confirm but on my end I think it's fine adding `cg_gcc` to the list.

Reply:
> Note that the "maintain" role shouldn't ideally be used for anything, it is not necessary for most use-cases. 

Interesting. Do you think then we should downgrade the role of T-compiler from `maintain` to `write` to make everything consistent?